med.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```

Comment:
Is this file related to this PR? I don't think so

Reply:
Sir,  in the 'services' tab, I observed that for placing orders, the previous developers had mentioned a section called "med.html", which didn't lead anywhere. So to fix this, I created this page, facilitating proper routing.

Reply:
@Nidhi-Satyapriya Good to keep it in then